src/uct/api/uct.h

```diff
@@ -322,8 +325,9 @@ enum {
                                               remote memory key for remote memory
                                               operations */
     UCT_MD_FLAG_ADVISE    = UCS_BIT(4),  /**< MD support memory advice */
-    UCT_MD_FLAG_FIXED     = UCS_BIT(5)   /**< MD support memory allocation with
+    UCT_MD_FLAG_FIXED     = UCS_BIT(5),  /**< MD support memory allocation with
                                               fixed address */
+    UCT_MD_FLAG_REG_NC    = UCS_BIT(6)   /**< MD support non-contig registration */
```

Comment:
why need a flag in memory domain, since no operation/function is added to the memory domain?